### DISCUSSION TITLE
Option for changing the keybinding for accepting the suggestions

### DISCUSSION THREAD

**Discussion by neilsabde** [community/community#6919] [Copilot Conversations] [answered]
It would be great if there was an option to change the keybinding for accepting the suggestions. I would like to change the Tab keybinding for completion in vim to something else because many people use the tab binding for opening the completion menu. Thanks in advance!

**Comment by tpope** [comment, marked as answer]
You're in luck, this is supported (but not yet documented).

```vim
inoremap <silent><expr> <C-]> copilot#Accept("")
let g:copilot_no_tab_map = 1
```

The part in parentheses is the fallback when there's nothing to accept. An empty string does nothing.

**Reply by wakywayne** [reply]
how do you edit the key binding for declining the inline expression? when the inline text is showing I cannot get my typescript auto complete which is more reliable for object key values

**Comment by neilsabde** [comment]
Thank you so much! 😀

**Comment by unphased** [comment]
~I'm not able to use this method to bind to `<CR>`. Is this a limitation? What other maps are verboten?~ I think i am finding out that this is because of another plugin that has `imap`'d `<CR>`...

Also, both the posted and the method given in the `:h copilot` work, the latter is: 

```
      imap <silent><script><expr> <C-J> copilot#Accept("\<CR>")
      let g:copilot_no_tab_map = v:true
```

Not sure what the functional difference is between the two (aside from the default mapped key being also different). Presumably the more complicated looking one has better behaviors?
